src/main/java/nl/knaw/dans/easy/sword2examples/SimpleDeposit.java

```diff
@@ -34,7 +34,7 @@
      * Sends a bag to the easy-sword2 service and tracks its status until it is archived or failure is reported.
      * 
      * @param args
-     *        0. zipped bag to send, 1. collection URL (Col-IRI), 2. EASY user name, 3. EASY password
+     *        1. collection URL (Col-IRI), 2. EASY user name, 3. EASY password, 0. zipped bag to send,
```

Comment:
did the order of the arguments change, or is this an accidental change?

Reply:
The order in the block comment did not match the order in which arguments were actually parsed. 
This was mentioned in issue: https://github.com/DANS-KNAW/easy-sword2-dans-examples/issues/16
I will add a small modification to this, the current order of args should remain. the same, but the numbers should run from 0-3.